rcl/test/rcl/test_logging_rosout.cpp

```diff
@@ -134,12 +130,49 @@ class TEST_FIXTURE_P_RMW (TestLoggingRosoutFixture)
     EXPECT_EQ(RCL_RET_OK, rcl_logging_fini()) << rcl_get_error_string().str;
   }
 
-protected:
+  virtual void call_rcl_init()
+  {
+    rcl_ret_t ret = rcl_init(0, NULL, &init_options_, this->context_ptr);
+    ASSERT_EQ(RCL_RET_OK, ret) << rcl_get_error_string().str;
+  }
+
+  virtual void update_node_option() {}
+
+  rcl_init_options_t init_options_;
+  rcl_node_options_t node_options_;
```

Comment:
I'll suggest to remove the trailing `_`, just to be consistent with the rest.  We aren't totally consistent in tests, but we may as well be consistent within the same class.